convert/src/vertex_to_bd/badgerdoc_format/plain_text_converter.py

```diff
@@ -31,14 +88,15 @@ def __init__(
         self.font_width = font_width
         self.line_spacing = line_spacing
 
-    def convert(self, text: str) -> Page:
-        paragraphs = self._get_paragraphs(text)
+        self.text_wrapper = TextWrapper(line_length=self.line_tokens_amount)
 
+    def convert(self, text: str) -> Page:
+        paragraphs = text.splitlines(keepends=True)
```

Comment:
Do we need these paragraphs, while TextWrapper can work with "\n"?

Reply:
Good point! Fixed